pkg/models/live.go

```diff
@@ -15,12 +15,18 @@ type ChannelHandler interface {
 
 	// Called when something writes into the channel.  The returned value will be broadcast if len() > 0
 	OnPublish(c *centrifuge.Client, e centrifuge.PublishEvent) ([]byte, error)
+
+	// Called from the HTTP API
+	DoChannelHTTP(c *ReqContext, channel string)
 }
 
-// ChannelHandlerProvider -- this should be implemented by any core feature
-type ChannelHandlerProvider interface {
+// ChannelNamespaceHandler -- this should be implemented by any core feature
```

Comment:
Documentation comments should be sentences.
```suggestion
// ChannelNamespaceHandler should be implemented by any core feature.
```